### ISSUE TITLE
JetEngine Profile Builder Custom menu links

### ISSUE THREAD

**Issue by edgBlw** [CrocoBlock/suggestions#5080]
Also currently we cant enter custom links inside the profile menu. It'd be great if can do so. 
That in the profile option istead of a page, we can add our custom liks in the menu, like logout, external website...

**Comment by nreljed** [comment]
the best example of what is possible with links and custom icons
here in example rankmath account page (it's just a WooCommerce account) but note the possibility of putting links and adjusting icons in a precise way

![demo](https://user-images.githubusercontent.com/82481592/160266288-88129237-89fc-4a68-aa2c-b8d086779984.gif)


Regarding the logout it is perhaps preferable to include it as an "add logout action" option for logical and especially dynamic reasons (site url + /wp-login.php?action=logout)

either in 
Profile Builder>account page

![Capture d’écran 2022-03-27 051203](https://user-images.githubusercontent.com/82481592/160266358-19c61fb2-e9a1-4483-9e33-e467f93ccfc1.png)

or in profile menu widget

![Capture d’écran 2022-03-27 050343](https://user-images.githubusercontent.com/82481592/160266369-28ba9e51-6d2e-490b-8953-d23c0961070e.png)

**Comment by vovkasolovev** [comment]
I agree, these features are necessary. Especially the addition of custom URLs, such a feature would close a huge part of the needs and reduce the number of templates.

Faced with this, when the site needed to have both Profile builder and WooCommerce cabinet, which has its own My Account pages. Since we have JetWooBuilder, it would be great to have a single Account pages instead of two separate ones (I had to make them look the same).

It would also be great if Profile Builder would somehow make use of JetMenu features, such as icons and badges. And not, as it is now, an incomplete duplication of WordPress menu page features.

Profile builder is a great feature that lacks a bit of unity with other super plugins.
